frontend/src/main.js

```diff
@@ -1,22 +1,21 @@
 import '@bcgov/bc-sans/css/BCSans.css'
 
-import { createApp } from 'vue'
-
-import { library } from '@fortawesome/fontawesome-svg-core'
-import { faCheckCircle, faChevronDown, faExclamationCircle, faExclamationTriangle, faInfoCircle, faSpinner, faTimes, faQuestionCircle } from '@fortawesome/free-solid-svg-icons'
-import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { faCheckCircle, faChevronDown, faExclamationCircle, faExclamationTriangle, faInfoCircle, faPlus, faQuestionCircle, faSpinner, faTimes, faTrashAlt } from '@fortawesome/free-solid-svg-icons'
```

Comment:
I don't think we are using faPlus. To improve performance and minimize size we don't want to include anything that's unused.